org.eclipse.xtext/META-INF/MANIFEST.MF

```diff
@@ -54,17 +41,7 @@ Export-Package: org.eclipse.xtext,
    org.eclipse.xtext.tests",
  org.eclipse.xtext.formatting2.internal;x-friends:="org.eclipse.xtext.tests,
    org.eclipse.xtend.core",
- org.eclipse.xtext.formatting2.regionaccess;x-friends:="org.eclipse.xtext.ide,
-   org.eclipse.xtext.ide.tests,
```

Comment:
intended?

Reply:
Yes. Not sure why github display it this way, but I removed the x-friends clause only. This marks the API as final and removes the warnings for formatters impolemented the new API. Have just double checked that I really removed the x-friends clause only.